src/Helpers/PageHelpers/PageHelper.cs

```diff
@@ -96,6 +106,28 @@ public class PageHelper : IPageHelper
             return formModel;
         }
 
+        private async Task AddDynamicOptions(IElement element, FormAnswers formAnswers)
+        {
+            var submitDetails = element.Properties.Lookup
+                .SingleOrDefault(x => x.EnvironmentName.Equals(_environment.EnvironmentName));
+
+            Answers query = formAnswers.AllAnswers
+                .SingleOrDefault(x => x.QuestionId.Equals(element.Properties.LookupQuestionIdQueryKey));
```

Comment:
can this be changed to support multiple querystring, currently will only support one. There is a method within ActionHelper.GenerateUrl which will do this for u.